helidon-cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -90,22 +105,42 @@ public String toString() {
         GRADLE,
     }
 
+    static final String DEFAULT_FLAVOR = "SE";
+    static final String DEFAULT_APPTYPE = "basic";
+    static final String DEFAULT_GROUP_ID = "mygroupid";
+    static final String DEFAULT_ARTIFACT_ID = "myartifactid";
+    static final String DEFAULT_PACKAGE = "mypackage";
+    static final String DEFAULT_NAME = "myproject";
+
     @Creator
     InitCommand(
-        CommonOptions commonOptions,
-        @KeyValue(name = "flavor", description = "Helidon flavor", defaultValue = "SE") Flavor flavor,
-        @KeyValue(name = "build", description = "Build type", defaultValue = "MAVEN") Build build,
-        @KeyValue(name = "version", description = "Helidon version") String version,
-        @KeyValue(name = "groupid", description = "Project's group ID") String groupId,
-        @KeyValue(name = "artifactid", description = "Project's artifact ID") String artifactId,
-        @KeyValue(name = "package", description = "Project's package name") String packageName) {
+            CommonOptions commonOptions,
+            @Flag(name = "batch", description = "Enables non-interactive mode") boolean batch,
+            @KeyValue(name = "flavor", description = "Helidon flavor",
+                    defaultValue = DEFAULT_FLAVOR) Flavor flavor,
+            @KeyValue(name = "build", description = "Build type",
+                    defaultValue = "MAVEN") Build build,
+            @KeyValue(name = "version", description = "Helidon version") String version,
+            @KeyValue(name = "apptype", description = "Application type",
```

Comment:
Again, we can change `--apptype` to `--type` later.

Reply:
Let's do this later, `apptype` is everywhere and changing it just here seems wrong.

Reply:
Totally agree, let's get this in.